### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by rishabh486**
Screenshot (47)
pr review for homepage created

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+const nextButton = document.querySelector(".carousel__button--right");
+const prevButton = document.querySelector(".carousel__button--left");
+const dotsNav = document.querySelector(".carousel__nav");
+const dots = Array.from(dotsNav.children);
+
+const slideWidth = slides[0].getBoundingClientRect().width;
+
+// slides[0].style.left = 0 + "px";
+// slides[1].style.left = slideWidth * 1.5+ "px";
+// slides[2].style.left = slideWidth * 2 + "px";
```
Comment (DivyanshSareen) [inline app.js:12]: remove comment if not intended for future use

File: index.css [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+a {
+  font-family: Arial, Helvetica, sans-serif;
+  text-decoration: none;
+  color: black;
+}
+.navbar {
+  width: 100%;
+  height: 8rem;
+  background-color: #49a0eb;
```
Comment (DivyanshSareen) [inline index.css]: var for colors

File: index.css [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  justify-content: flex-end;
+}
+.navbar .rightside input {
+  width: 25rem;
+  height: 3.5rem;
+}
+.navbar .rightside button {
+  height: 3.65rem;
+  width: 5rem;
+  background-color: #63edff;
```
Comment (DivyanshSareen) [inline index.css]: var for color

File: index.css [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  height: 5rem;
+}
+.icon-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 4rem;
+  height: 4rem;
+  color: #333333;
+  background: lavenderblush;
```
Comment (DivyanshSareen) [inline index.css]: use var here as well

**Review by DivyanshSareen — APPROVED** [review]
UI is clean, code is readable. Can use variables for color codes

File: index.css [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  width: 5rem;
+  height: 5rem;
+}
+.icon-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 4rem;
+  height: 4rem;
+  color: #333333;
```
Comment (developer-anand) [inline index.css]: use var for colors.

File: index.css [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+  position: relative;
+  height: 35rem;
+  width: 91.3rem;
+}
+.carousel__image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
```
Comment (developer-anand) [inline index.css:698]: be consistent with spacing.

**Review by developer-anand — APPROVED** [review]
Good to merge with changes.

**Pull request merged**